src/coreclr/inc/contract.h

```diff
@@ -1483,7 +1482,7 @@ typedef __SafeToUsePostCondition __PostConditionOK;
 // to return and we need to ensure that we don't allow a return where one should not happen
 //
 #define RETURN                                                                              \
-    while (DEBUG_ASSURE_SAFE_TO_RETURN, TRUE)                                               \
+    while (TRUE)                                                                            \
```

Comment:
What's the consequence of using wrong `return` macro with contracts?

Reply:
You get an "unused goto label" error when you use `return` instead of `RETURN` with a contract that checks the return value.